selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/RolloutUpdateStrategyWithWorkspacesStartStopTest.java

```diff
@@ -0,0 +1,97 @@
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RolloutUpdateStrategyWithWorkspacesStartStopTest {
+
+  private static final String COMMAND_FOR_GETTING_CURRENT_DEPLOYMENT_CHE =
+      "get dc | grep che | awk '{print $2}'";
+
+  @InjectTestWorkspace(startAfterCreation = false)
+  private TestWorkspace workspaceForStarting;
+
+  @Inject private TestWorkspace workspaceForStopping;
+  @Inject private TestWorkspaceServiceClient testWorkspaceServiceClient;
+  @Inject private WebDriverWaitFactory webDriverWaitFactory;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private CheTestSystemClient cheTestSystemClient;
+  @Inject private OpenShiftCliCommandExecutor openShiftCliCommandExecutor;
+  @Inject private Dashboard dashboard;
+  @Inject private Workspaces workspaces;
+
+  @Test
+  public void createMavenArchetypeStartProjectByWizard() throws Exception {
+    int currentRevision = getRevision();
+
+    // open 'Workspaces' page
+    dashboard.open();
+    dashboard.waitDashboardToolbarTitle();
+    dashboard.selectWorkspacesItemOnDashboard();
+
+    // check existing of expected workspaces and their statuses
+    workspaces.waitPageLoading();
+    workspaces.waitWorkspaceIsPresent(workspaceForStopping.getName());
+    workspaces.waitWorkspaceIsPresent(workspaceForStarting.getName());
+
+    workspaces.waitWorkspaceStatus(workspaceForStopping.getName(), Workspaces.Statuses.RUNNING);
+    workspaces.waitWorkspaceStatus(workspaceForStarting.getName(), Workspaces.Statuses.STOPPED);
+
+    // execute rollout command
+    executeRolloutUpdateCommand();
+
+    // execute stop-start commands for existing workspaces
+    Assert.assertEquals(cheTestSystemClient.getStatus(), SystemStatus.RUNNING);
+    workspaces.clickOnWorkspaceStopStartButton(workspaceForStarting.getName());
+    workspaces.clickOnWorkspaceStopStartButton(workspaceForStopping.getName());
+
+    // wait successful results of the stop-start requests
+    workspaces.waitWorkspaceStatus(workspaceForStopping.getName(), Workspaces.Statuses.STOPPED);
+    workspaces.waitWorkspaceStatus(workspaceForStarting.getName(), Workspaces.Statuses.RUNNING);
+
+    // check that che is updated
+    waitRevision(currentRevision + 1);
+  }
+
+  private int getRevision() {
+    try {
+      return Integer.parseInt(
+          openShiftCliCommandExecutor.execute(COMMAND_FOR_GETTING_CURRENT_DEPLOYMENT_CHE));
+    } catch (IOException ex) {
+      throw new RuntimeException(ex.getLocalizedMessage(), ex);
+    }
+  }
+
+  private void waitRevision(int expectedRevision) {
+    webDriverWaitFactory
+        .get(100)
+        .until((ExpectedCondition<Boolean>) driver -> expectedRevision == getRevision());
+  }
+
+  private void executeRolloutUpdateCommand() throws Exception {
+    openShiftCliCommandExecutor.execute("rollout latest che");
```

Comment:
`"rollout latest che"` would be the constant for the both classes: **RolloutUpdateStrategyWithWorkspacesStartStopTest** and **RolloutUpdateStrategyWithEditorTest**